src/components/TableColumnHeader.vue

```diff
@@ -1,15 +1,14 @@
 <template>
-    <th
-        @click="clicked"
-        :class="headerClass"
-        role="columnheader"
-        slot-scope="col"
-        :aria-sort="ariaSort"
-        :aria-disabled="ariaDisabled"
-        v-if="this.isVisible"
-    >
-        {{ label }}
-    </th>
+  <th
+    @click="clicked"
+    :class="headerClass"
+    role="columnheader"
+    :aria-sort="ariaSort"
+    :aria-disabled="ariaDisabled"
+    v-if="this.isVisible"
```

Comment:
The order of props seems arbitrary.

Refer to CBX vue styleguide for props order

Reply:
indeed it seems arbitrary